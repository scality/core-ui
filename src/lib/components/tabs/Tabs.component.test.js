import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/tabs/__snapshots__"
});
