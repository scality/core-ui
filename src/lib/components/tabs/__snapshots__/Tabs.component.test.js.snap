// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Tabs Default 1`] = `
<div
  className="sc-EHOje dZmSdl"
>
  <h3
    className="sc-bZQynM dIEjhl"
  >
    Default Tabs 
  </h3>
  <div
    className="sc-bdVaJa iROZJK sc-tabs"
  >
    <div
      className="sc-bwzfXH ihlkSE sc-tabs-bar"
    >
      <div
        className="sc-htpNat jSgFvj sc-tabs-item"
        onClick={[Function]}
        selected={true}
      >
        <p
          className="sc-bxivhb cDojnC sc-tabs-item-title"
          selected={true}
        >
          Detail
        </p>
      </div>
      <div
        className="sc-htpNat iHwDjt sc-tabs-item"
        onClick={[Function]}
        selected={false}
      >
        <p
          className="sc-bxivhb abRhr sc-tabs-item-title"
          selected={false}
        >
          Pods
        </p>
      </div>
      <div
        className="sc-htpNat iHwDjt sc-tabs-item"
        onClick={[Function]}
        selected={false}
      >
        <p
          className="sc-bxivhb abRhr sc-tabs-item-title"
          selected={false}
        >
          Volumes
        </p>
      </div>
    </div>
    <div
      className="sc-ifAKCX jGsdUL sc-tabs-item-content"
    />
  </div>
</div>
`;
